addons/web_tour/static/src/js/tip.js

```diff
@@ -38,12 +38,13 @@ var Tip = Widget.extend({
                 x: 50,
                 y: 50,
             },
+            scrollContent: _t('Scroll to reach the next step.'),
         });
         this.position = {
             top: "50%",
             left: "50%",
         };
-        this.viewPortState = 'in';
+        this.viewPortState = null;
```

Comment:
Is it a problem to have it initialized as 'in'? if yes at least use undefined instead of null. null should be used for variables supposed to contain an object, here it is a string.

Reply:
Yes, when the anchor is out of the viewport at the beginning. Ok !

Reply:
Well at the beginning it should not be reposition while the viewPortState is not computed..

Reply:
Sorry I lost myself, I did it because the part of the code when I add "o_tooltip_parent" is not called if the anchor is in the viewport at the beginning. I'll just move this part of the code in another function, and call it once at the beginning and once every time I move the tip.
I'll put back "in"